### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by TechandEco**
## Description

Updated README

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by TechandEco** [comment]
Can you please review this @davidcavazos

File: people-and-planet-ai/image-classification/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AutoML image classification
 
-1. 🎬 Watch the introduction video in Youtube!
+1. 🎬 Watch the 8min introduction video on Youtube!
 
    [![Watch the video in YouTube](http://img.youtube.com/vi/hUzODH3uGg0/0.jpg)](https://youtu.be/hUzODH3uGg0 "Watch the video in YouTube")
```
Comment (parthea) [inline people-and-planet-ai/image-classification/README.md:5]:
```suggestion
   [![Watch the video in YouTube](http://img.youtube.com/vi/hUzODH3uGg0/0.jpg)](https://youtu.be/hUzODH3uGg0 "Watch the video on YouTube")
```

**Review by parthea — COMMENTED** [review]
Just one observation, otherwise LGTM. @TechandEco Please can you sign the CLA?

**Review by dinagraves — APPROVED** [review]

**Comment by parthea** [comment]
Hi @TechandEco,

Please can you sign the CLA?

**Comment by TechandEco** [comment]
> Hello,


I was out of office. I have signed it my friend.

**Comment by dandhlee** [comment]
@TechandEco could you reply with:
`@googlebot I signed it!`
then it will complete the CLA process :D

**Review by davidcavazos — APPROVED** [review]

**Comment by davidcavazos** [comment]
Thanks!

**Comment by dandhlee** [comment]
we need @TechandEco to echo to CLA bot to be able to merge this PR.

**Comment by TechandEco** [comment]
@googlebot I signed it!

**Comment by dandhlee** [comment]
@TechandEco under your username "TechandEco" I don't see a CLA signed, did you sign a corporate CLA by chance?

**Comment by parthea** [comment]
I'm going to close PR due to inactivity but please feel free to re-open it.

**Pull request closed without merging**